File: Python/1_Iniciante/1072.py
n = int(input())
valIn = 0
valOut = 0

for i in range (n):
    a = int(input())
    if(a>=10 and a<=20):
        valIn+=1
    else:
        valOut+=1

print(f'{valIn} in')
print(f'{valOut} out')

File: Ruby/1_Iniciante/1072.rb
n = gets.to_i
valIn = 0
valOut = 0

n.times do
    a = gets.to_i
    if a >= 10 and a <= 20
        valIn+=1
    else
        valOut+=1
    end
end

puts "#{valIn} in"
puts "#{valOut} out"
